File: projects/src/desktop_client/lib/plugins/client.core/client/keyboard.js
exports.plugin = function(router) {

	return {
		event: function(code, bScan, dwFlags) {
			console.log('keyboard event: %d, %d, %d', code, bScan, dwFlags);
			router.desktop.keyboardEvent(code, bScan, dwFlags);
		}
	}
}

File: wkm/src/screens/keyboard.cpp
#include "screens/keyboard.h"
#include "screens/screens.h"

namespace Screens
{
	Keyboard::Keyboard(Screen* screen):
	_screen(screen)
	{

	}

	bool Keyboard::event(int bvk, int bScan, int dwFlags)
	{

		//this->_screen->focus();

		keybd_event(bvk, bScan, dwFlags, 0);

		return true;
	}
}